### DISCUSSION TITLE
Tutor is a Blank File

### DISCUSSION THREAD

**Discussion by theduke** [helix-editor/helix#10252] [Troubleshooting]
### Summary

I started `:tutor` once, and it worked fine.

Then I closed helix and at some point tried `:tutor` again.
Now all I get is a blank file.

Installed with `cargo install --path` directly from a repo clone.

### Platform

Linux

### Terminal Emulator

alacritty

### Helix Version

helix 22.12 (96ff64a8)

**Comment by theduke** [comment]
Same result when I run `hx --tutor`.

**Comment by kirawi** [comment]
You also need to link your runtime folder: https://docs.helix-editor.com/install.html#build-from-source

**Comment by theduke** [comment]
Ah.
But then why did it work the first time?

Probably the `:tutor` command should print out an error if the tutor file can not be found.

**Comment by kendfss** [comment]
for some reason my default runtime directory points to `.cargo/bin/runtime`, which does not exist, even though the docs say to use `.config/helix/runtime`. I've overwritten the $HELIX_RUNTIME variable but am wondering if should this be raised as a separate issue?

**Comment by janhrastnik** [comment]
In my case I had an obsolete `.config/helix/runtime` directory, with a `tutor.txt` file, but since commit c2e4108, helix searches for a `tutor` file instead, without the .txt suffix. My helix is built from source, if I install it in a different way, then I don't get this issue as it then finds an appropriate runtime directory to use.
